From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Shelley Vohr <shelley.vohr@gmail.com>
Date: Tue, 16 Jan 2024 19:39:10 +0100
Subject: test: match wpt/streams/transferable/transform-stream-members.any.js
 to upstream

All four of this calls should fail - see third_party/blink/web_tests/external/wpt/streams/transferable/transform-stream-members.any-expected.txt

diff --git a/test/wpt/status/streams.json b/test/wpt/status/streams.json
index af3646c65ea6609e17638da925f86de8c9d6221c..f1d9e76718c264aafb9c27c516589d9874d8b73f 100644
--- a/test/wpt/status/streams.json
+++ b/test/wpt/status/streams.json
@@ -53,7 +53,9 @@
     "fail": {
       "expected": [
         "Transferring [object TransformStream],[object ReadableStream] should fail",
-        "Transferring [object TransformStream],[object WritableStream] should fail"
+        "Transferring [object TransformStream],[object WritableStream] should fail",
+        "Transferring [object ReadableStream],[object TransformStream] should fail",
+        "Transferring [object WritableStream],[object TransformStream] should fail"
       ]
     }
   },
